starcheck/calc_ccd_temps.py

```diff
@@ -153,7 +153,7 @@ def get_ccd_temps(oflsdir, outdir='out',
                 '######################################\n')
 
     # load more general starcheck characteristics to get red/yellow limits
-    char = yaml.load(open(char_file))
+    char = yaml.load(open(char_file), Loader=yaml.SafeLoader)
```

Comment:
Generally speaking I would go for `yaml.safe_load(..)` as a generic replacement, but this is also fine.